router/core/router.go

```diff
@@ -1116,45 +1117,36 @@ func (r *Router) Start(ctx context.Context) error {
 		}()
 
 		if r.executionConfig != nil && r.executionConfig.Watch {
+			go watcher.LogSimpleWatch(ctx, watcher.SimpleWatcherOptions{
```

Comment:
The old watcher is unused now, correct? Any reason to keep unused code?

Reply:
Posterity I guess, but its broken in other ways anyway. I'll remove it